test/integration/consultation_test.rb

```diff
@@ -87,21 +95,36 @@ class ConsultationTest < ActionDispatch::IntegrationTest
     end
   end
 
-  test "consultation outcome documents render" do
+  test "renders consultation outcome attachments (as-is and directly)" do
     setup_and_visit_content_item("consultation_outcome")
 
+    assert page.has_text?("Download the full outcome")
     within '[aria-labelledby="download-the-full-outcome-title"]' do
       assert page.has_text?("Employee Share Schemes: NIC elections - consulation response")
     end
+
+    setup_and_visit_content_item("consultation_outcome_with_featured_attachments")
```

Comment:
Each of these `test` blocks seem like they should be split into two `test` blocks - one for `as-is`, and one for `directly`. I appreciate there's no other way of grouping them together, save for a comment in between every two blocks, but it would make it easier to delete the `as-is` blocks when we come to deprecate that behaviour.

Reply:
While I agree they could be split up, I think that would decrease the readability of the rest of the test file, as there is indeed no way of grouping them, and they are effectively testing the same thing. One of the commit messages says this, somewhere.

Reply:
I don't think I would find it that much harder to delete the lines within the test, when we (finally) disable the legacy functionality.